crates/kitsune_p2p/proxy/benches/thru.rs

```diff
@@ -1,234 +1,238 @@
-use criterion::{/*black_box,*/ criterion_group, criterion_main, Criterion};
```

Comment:
@maackle - The whole proxy crate should be deleted, proxy is only used by tx2. The unfortunate exception at the moment is the ProxyUrl type, which you could move into the tx_utils module if you don't want to factor that whole thing out of the code yet which I imagine would be a decent undertaking.